packages/interactivity/src/store.ts

```diff
@@ -34,25 +34,25 @@ const deepMerge = ( target: any, source: any ) => {
 	}
 };
 
-const parseInitialState = () => {
+const parseInitialData = () => {
 	const storeTag = document.querySelector(
 		`script[type="application/json"]#wp-interactivity-data`
 	);
-	if ( ! storeTag?.textContent ) return {};
-	try {
-		const { state } = JSON.parse( storeTag.textContent );
-		if ( isObject( state ) ) return state;
-		throw Error( 'Parsed state is not an object' );
-	} catch ( e ) {
-		// eslint-disable-next-line no-console
-		console.log( e );
+	if ( storeTag?.textContent ) {
+		try {
+			return JSON.parse( storeTag.textContent );
+		} catch ( e ) {
+			// eslint-disable-next-line no-console
+			console.log( e );
```

Comment:
Do we want a console.log here?
```suggestion
			return {};
```

Reply:
I added a comment, as we already have a return at the end of the function. https://github.com/WordPress/gutenberg/pull/58749/commits/eb0ea96fc4e132178544315b8168923fd4a2d978